### DISCUSSION TITLE
I am having some trouble using this library (nodeJS environment)

### DISCUSSION THREAD

**Discussion by bkhalil4797** [opensource9ja/danfojs#290] [Q&A]
Hi everyone,
I just tried this library but i am having trouble to make it work
here my file

`const dfd = require("danfojs-node");

async function readCSV() {
  const df = await dfd.read_csv("file://../../extra/titanic data/train.csv");
  // Print the columns
  df.head().print();
  // Describe the DataFrame
  df.describe().print();
}

// Demonstrate
readCSV();
`
And i get this error when i try to run this code 
node .\1_readprint.js
2021-09-28 15:49:05.184095: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.

can someone help me solve this and why this is happening

from package.json
  "dependencies": {
    "danfojs-node": "^0.3.0"
  }

from package-lock.json
"@tensorflow/tfjs": {
      "version": "3.6.0",
"@tensorflow/tfjs-backend-cpu": {
      "version": "3.6.0",
"@tensorflow/tfjs-backend-webgl": {
      "version": "3.6.0",
"@tensorflow/tfjs-backend-webgl": {
      "version": "3.6.0",
"@tensorflow/tfjs-converter": {
      "version": "3.6.0",
"@tensorflow/tfjs-data": {
      "version": "3.6.0",
"@tensorflow/tfjs-layers": {
      "version": "3.6.0",
"@tensorflow/tfjs-node": {
      "version": "3.6.1",
+ others

Spec
Processor	Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz   2.71 GHz
Installed RAM	12.0 GB (11.9 GB usable)
Edition	Windows 10 Home
OS build	19043.1237
node version : v14.18.0
npm version : 7.20.1

**Comment by risenW** [comment]
From the error message, it seems the package was not properly installed. Can you share more details like the version of the library you're using and your system spec

**Reply by bkhalil4797** [reply]
I am trying to make my own build of tensorflowJS following https://www.npmjs.com/package/@tensorflow/tfjs-node Section Build optimal TensorFlow from source to see if it solve it (I updated my post)
